style.css

```diff
@@ -0,0 +1,94 @@
+* {
+    margin: 0;
+    border: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #eee;
+}
+
+#header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 1rem 2rem;
+    position: fixed;
+    width: auto;
+    background-color: #eee;
+}
+
+#header>img {
+    width: 30%;
+}
+
+#features >div {
+    margin: 2rem;
+}
+
+#nav-bar>ul {
+    list-style: none;
+    display: flex;
+    gap: 2rem;
+}
+
+.nav-link {
+    text-decoration: none;
+}
+
+footer {
```

Comment:
Kindly consider this style and the one commented on below as they share the same attribute and values. While this works, it is recommended that you group all targeted elements with similar styling to avoid repetitions. Please consider grouping them together.